src/CommunityToolkit.Maui.Markup/AppBuilderExtensions.cs

```diff
@@ -5,13 +5,35 @@
 /// </summary>
 public static class AppBuilderExtensions
 {
+	static IServiceProvider serviceProvider =>
+#if WINDOWS
+		MauiWinUIApplication.Current.Services;
+#elif ANDROID
+		MauiApplication.Current.Services;
+#elif IOS || MACCATALYST
+		MauiUIApplicationDelegate.Current.Services;
+#elif TIZEN
+		throw new NotImplementedException();
+#else
+		throw new InvalidOperationException($"{nameof(IServiceProvider)} requires a platform specific implementation");
+#endif
+
 	/// <summary>
 	/// Initializes the .NET MAUI Community Toolkit Markup Library
 	/// </summary>
 	/// <param name="builder"><see cref="MauiAppBuilder"/> generated by <see cref="MauiApp"/> </param>
 	/// <returns><see cref="MauiAppBuilder"/> initialized for <see cref="CommunityToolkit.Maui.Markup"/></returns>
 	public static MauiAppBuilder UseMauiCommunityToolkitMarkup(this MauiAppBuilder builder)
 	{
+#if DEBUG
```

Comment:
Haha - whoops! 

My intent is that this will only be wired-up when the user's app is in the DEBUG configuration. But you're totally right - this is _our_ DEBUG config which doesn't ship with the nugget package.

I'll dig into this to figure out a way that we can ensure that this code ships _and_ that it is not included in user's RELEASE apps.

Reply:
If you look into the source code for System.Diagnostics.Debugger.Debug.WriteLine you will they use an attribute that I can't remember right now. Maybe that is what we want.

Reply:
^@GeorgeLeithead Pedro was one step ahead of ya 😁

Reply:
Good ole @pictos .  I was meaning the Conditional attribute, but it also looks like you two have that covered :-D @brminnick